en-US/firefox/accounts-2018.lang

```diff
@@ -13,6 +14,12 @@ Get a Firefox Account – Keep your data private, safe and synced
 Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.
```

Comment:
the original PR missed this one `log-in` to `login`

Reply:
D'oh. I'll add the string here now and submit a PR to update it in the template behind the same tag. Good catch!